app/server/models/organisation.py

```diff
@@ -156,7 +160,7 @@ def __init__(self, token=None, is_master=False, **kwargs):
     
         self.external_auth_username = 'admin_'+ self.name.lower().replace(' ', '_')
         self.external_auth_password = secrets.token_hex(16)
-
+        self.valid_roles = list(ACCESS_ROLES.keys())
```

Comment:
suggest changing the init to
```
def __init__(self, token=None, is_master=False,  valid_roles=None,**kwargs):
```
and this line to
```
self.valid_roles = valid_roles or list(ACCESS_ROLES.keys())
```

Otherwise if a user provides valid_roles as a kwarg it'lll be overwritten by this and the user will be confused